langchain-core/langchain.config.js

```diff
@@ -61,6 +61,7 @@ export const config = {
     "utils/tiktoken": "utils/tiktoken",
     "utils/types": "utils/types",
     "utils/beta_warning": "utils/beta_warning",
+    "utils/is_zod_schema": "utils/is_zod_schema",
```

Comment:
Feels wrong to give this its own entrypoint - I actually think adding it to `types` would be reasonable here.